### ISSUE TITLE
Disable firecamp from burning?

### ISSUE THREAD

**Issue by Rakso69** [Sh0wdown/UnofficialCrusaderPatch#403]
So annoying to create "maximum economy effectivness around stockpile" for the AI Villages due to this shit always in the way, which needs to be surrounded with wells all around in order to not catch fire, that often leaves AI dead for long.
Jeez, why didn't I thought of type this issue earlier? It's such a hassle.
![image](https://user-images.githubusercontent.com/45049934/59287411-e43e6c00-8c71-11e9-91f2-86887ab0cce6.png)

**Comment by Panbutt** [comment]
It's been discussed before iirc but yeah fire needs a serious nerf. Sometimes fire can just jump through walls and moats and catch everything on fire. It's pretty broken imo.

**Comment by Truetobi91** [comment]
You just need enough wells, kappa.

**Comment by Russianstory** [comment]
So, in order to save space, it would be possible to make landfills and shooting ranges for the barracks less, or as for the engineers' guild. This will reduce the space and give a small bonus. Also, the Lord's Citadel should be as long as two warehouses.
=
Вот для экономии места можно было бы сделать полигоны и стрельбища у казарм меньше или же как у гильдии инженеров. Это сократит место и даст небольшой бонус. Так же Цитадель Лорда должна быть в длину как два склада.

**Comment by Rakso69** [comment]
Guys, here I speak only about disable firecamp from catch fire, not about whole fire mechanism in general. ;)

**Comment by Krarilotus** [comment]
I woud say, the firecamp burn saved multiple AI lords from finisher attacks. Just make sure, that your buildings dont touch it too much, and it ll be easily handled by a single well.Because only if they touch, the fire spreads instantly!

**Comment by Rakso69** [comment]
@Krarilotus Yeah... Makes game pointlessly longer for the mindless attacker too. I saw AI plenty times enter a burning base just because it wen't hole-open.
